sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -727,3 +727,71 @@ def get_random_bytes(self, count, **kwargs):
         parameters = self._models.GetRandomBytesRequest(count=count)
         result = self._client.get_random_bytes(vault_base_url=self._vault_url, parameters=parameters, **kwargs)
         return RandomBytes(value=result.value)
+
+    @distributed_trace
+    def get_key_rotation_policy(self, name, **kwargs):
+        # type: (str, **Any) -> Optional[KeyRotationPolicy]
+        """Get the rotation policy of a Key Vault key.
+
+        :param str name: The name of the key.
+
+        :return: The key rotation policy, or None if there is no policy.
+        :rtype: ~azure.keyvault.keys.KeyRotationPolicy or None
+        """
+        policy = self._client.get_key_rotation_policy(vault_base_url=self._vault_url, key_name=name, **kwargs)
+        if policy.id:
```

Comment:
I realize now that I originally had this in both the issue and my JS PR, but after some discussions with the team we agreed that it would not be necessary. There is a concept of a "default" rotation policy which means that we can always expect to get data back. 

Now, it might not have an id (if the user has never set one) which means that we need to model this differently (i.e. id is optional, as are the timestamps)... this was impactful for TypeScript, but might not be necessary for Python?

At any rate, what we're doing now is always return a KeyRotationPolicy, never `undefined` (or an Optional[T]) - let me know if you want to discuss this.

The TL;DR is - assume returning `KeyRotationPolicy._from_generated(policy)` is always safe (as long as id, createdOn, and updatedOn are nullable)

Reply:
Got it! I remembered you mentioning that a default policy had been discussed but I wasn't sure if this was implemented yet. I'll make those updates!